Misc/NEWS.d/next/Tools/Demos/2019-10-02-09-48-42.bpo-38347.2Tq5D1.rst

```diff
@@ -0,0 +1 @@
+Allow Python scripts to contain a '-' in their filename when working recursively.
```

Comment:
You should mention pathfix somewhere. And this NEWS entry is outdated: it's no longer about "-", but any character.

Reply:
Done in a68ad4207100b365d00ffbb745592408d1d55a98 and 05c39a5dd793c45fb6c2f2cff62b947a9576b352